libraries/rush-lib/src/logic/pnpm/SplitWorkspaceGlobalPnpmfileShim.ts

```diff
@@ -21,6 +23,7 @@ import type {
   IWorkspaceProjectInfo
 } from './IPnpmfile';
 import type { IPnpmShrinkwrapYaml } from './PnpmShrinkwrapFile';
+import _ from 'lodash';
```

Comment:
It seems `lodash` is not used in this file? Can we remove it

Reply:
Weird, probably auto imported by VSCode snippet, will remove it.